src/CampaignsAdmin/CampaignDetails.tsx

```diff
@@ -24,14 +25,13 @@ export function CampaignDetails() {
    */
   const uploadImageToSupabase = async (file: File) => {
     const filename = `${uuidv4()}.jpg`;
-    const { uri, apiKey } = settings.supabase;
```

Comment:
`uploadImageToSupabase` should be an independent utility. Since it's not related to the campaign it can live on its own file